d/handy/source/app.d

```diff
@@ -3,11 +3,11 @@ import std.algorithm: startsWith;
 
 void main() {
     ServerConfig cfg = ServerConfig.defaultValues();
-    cfg.workerPoolSize = 5;
+    cfg.workerPoolSize = 16;
```

Comment:
why 16 ?

Reply:
I just found it is good to have around 2xCores. But I don’t know what is suggested value for this number

Reply:
then it might be interesting to use `totalCPU`, the results are made on a 8 core CPU, but it might change when running on cloud

Reply:
Agree. Update handy code with totalCPUs in last commit